src/public/modules/forms/services/submissions.client.factory.js

```diff
@@ -248,10 +260,13 @@ function SubmissionsFactory(
       secretKey,
     ) {
       // Helper function to kill an array of EncryptionWorkers
-      const killWorkers = (workerPool) => {
-        workerPool.forEach((worker) => worker.terminate())
+      const killWorkers = (pool) => {
+        pool.forEach((worker) => worker.terminate())
       }
```

Comment:
can help to move the `killWorkers` function out of the `downloadEncryptedResponses` function?